lib/junos-ez/utils/config.rb

```diff
@@ -169,7 +169,7 @@ def rollback!( rollback_id = 0 )
   
   def diff?( rollback_id = 0 )
     raise ArgumentError, "invalid rollback #{rollback_id}" unless ( rollback_id >= 0 and rollback_id <= 50 )    
-    got = ndev.rpc.get_configuration( :compare=>'rollback', :rollback=> rollback_id.to_s )
+    got = ndev.rpc.get_configuration( :compare => 'rollback', :rollback => rollback_id.to_s, :format => 'text' )

```

Comment:
@lucasmpb  why do we need to add format? What changed in the newer version of Junos?

Reply:
Sorry @vnitinv I should have explained myself. I added a description to the pull request.